code/client/src/components/SiderMenu.jsx

```diff
@@ -40,10 +30,10 @@ const SiderMenu = ({ ...args }) => {
     <Layout.Sider collapsible={width < 900} onCollapse={c => setCollapsed(c)} {...args}>
       {/* <Image src='/assets/harmony.svg' /> */}
       <Row justify='center'>
-        <Image preview={false} src={collapsed ? HarmonyIcon : HarmonyLogo} style={{ padding: collapsed ? 16 : 32 }} />
+        <Image preview={false} src={collapsed ? HarmonyIcon : HarmonyLogo} style={{ cursor: 'pointer', padding: collapsed ? 16 : 32 }} onClick={() => history.push('/')} />
       </Row>
-      {!collapsed && <Text>{config.appName} {config.version}</Text>}
-      {!collapsed && <Beta>Beta</Beta>}
+      {!collapsed && <Text><a href='https://docs.harmony.one/home/developers/wallets/1wallet' target='_blank'>{config.appName} {config.version}</a></Text>}
+      {!collapsed && <Row justify='center' style={{ marginBottom: 10 }}><Tag color='#2db7f5'>Beta</Tag></Row>}
```

Comment:
Can we use a dimmer color? e.g. #0094c0 from the Harmony logo gradient. It seems too bright and too standout compared to the rest.

Reply:
![image](https://user-images.githubusercontent.com/913871/123887968-5771b400-d907-11eb-970e-4bd4a69d3c95.png)

![image](https://user-images.githubusercontent.com/913871/123887987-63f60c80-d907-11eb-9efd-29a4a2174722.png)

Reply:
Good call. Done
![image](https://user-images.githubusercontent.com/205550/123897916-269a7a80-d919-11eb-8bfa-b7ac3a04a487.png)